lib/lob/resources/resource_base.rb

```diff
@@ -20,8 +20,8 @@ def find(resource_id)
         submit :get, resource_url(resource_id)
       end
 
-      def create(options = {})
-        submit :post, endpoint_url, options
+      def create(options={}, headers={})
+        submit :post, endpoint_url, options, headers
```

Comment:
Just to check, is idempotency something that users opt in to? Or do we expect all create requests to be idempotent?